README.md

```diff
@@ -12,26 +12,94 @@ Declare the library as a project dependency as follows:
 "uk.gov.hmrc" %% "ui-test-runner" % "x.x.x" % Test
 ```
 
-See an [example](https://github.com/hmrc/platform-test-example-ui-journey-tests/blob/main/project/Dependencies.scala).
+See an [example](https://github.com/hmrc/platform-test-example-ui-tests/blob/main/project/Dependencies.scala).
 
 ### Configuration
 
-- System property `browser` must be set in order to execute tests. Arguments `chrome`, `edge` and `firefox` are available.
-- System property `environment` must be set in order to execute tests. Arguments `local`, `dev`, `qa` and `staging` are typically available, but will depend on your project configuration. See an [example](https://github.com/hmrc/platform-test-example-ui-journey-tests/blob/main/src/test/resources/application.conf).
+- System property `browser` must be set to execute tests. Arguments `chrome`, `edge` and `firefox` are available.
+- System property `environment` must be set to execute tests. Arguments `local`, `dev`, `qa` and `staging` are typically available, but will depend on your project configuration. See an [example](https://github.com/hmrc/platform-test-example-ui-journey-tests/blob/main/src/test/resources/application.conf).
```

Comment:
https://github.com/hmrc/platform-test-example-ui-journey-tests/blob/main/src/test/resources/application.conf
to be replaced with
https://github.com/hmrc/platform-test-example-ui-tests/blob/main/src/test/resources/application.conf